core/http/src/cookies.rs

```diff
@@ -1,8 +1,8 @@
 use std::fmt;
 use std::cell::RefMut;
 
-use crate::Header;
 use cookie::Delta;
+use crate::Header;
```

Comment:
Superfluous

Reply:
Done. CI timed out for some reason on the Mac debug build, but everything else passed.